packages/react-server/src/ReactFlightHooks.js

```diff
@@ -78,6 +78,9 @@ export const Dispatcher: DispatcherType = {
   useCacheRefresh(): <T>(?() => T, ?T) => void {
     return unsupportedRefresh;
   },
+  useMemoCache(size: number): Array<any> {
+    return new Array(size);
```

Comment:
this is Flight so we don't care about memoization, but we do want the code to run.